department-of-veterans-affairs/va.gov-cms
# VACMS 6167: Remove user/request_access form and webform but keep TippyJS.

## Description

Relates to #6167.  

This was kind of an experience.  I wanted to use Tippy as a locally managed library rather than pull it from a CDN, but apparently that process is at least 10% smarter than me.  I'd be delighted to hear how this can be done that way, if anyone has any tips.  I tried to do it with a `bower-asset` package, but that actually just pulled in the TS source and did not seem to include a distributable.  IDK if pulling this from a CDN is a dealbreaker or not.  

Also I had like five false positives where I thought I'd figured it out but I actually hadn't.

## Testing done
Opened Tugboat, hovered over the toolbar thingy, saw a tooltip, rejoicesobbed.

## Screenshots
<img width="298" alt="Screen Shot 2021-08-25 at 2 22 24 PM" src="https://user-images.githubusercontent.com/1318579/130844732-fa790229-e900-443e-aa25-26500c9414be.png">


## QA steps

As user _uid_ with _user_role_
1. Open Tugboat
2. Hover over the toolbar thingy
3. Verify that you see the tooltip.

### Definition of Done


### Select Team for PR review


### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
